docs/quickstarts/nextjs.mdx

```diff
@@ -57,14 +57,10 @@ pnpm add @clerk/nextjs
 
 ### Set your environment variables
 
-1. If you don't have an `.env.local` file in the root directory of your Next.js project, create one now.
-2. Find your Clerk publishable key and secret key. If you're signed into Clerk, the `.env.local` snippet below will contain your keys. Otherwise:
-  - Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys) and select your application.
-  - In the navigation sidebar, select **API Keys**.
-  - You can copy your keys from the **Quick Copy** section.
-3. Add your keys to your `.env.local` file.
+1. If you're signed into Clerk, the `.env.local` snippet below will contain your keys. Otherwise, get your API keys from your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys).

```

Comment:
If the users aren't signed in to their dashboard, that link will not take them to the desired page. So we need to have copy that tells users which page the API keys lives on. I've grabbed the language that we use on other pages, and inserted it into the suggestion.

Also, iirc, your previous suggestion was to refrain from using "below" because it relies on visual positioning, which may not be universally understood by all users. Blind or visually impaired users who rely on screen readers, for example, may not perceive the spatial relationship implied by "below." I've suggested we use "following" instead !

```suggestion
1. If you are signed into Clerk, the following `.env.local` snippet will contain your keys. Otherwise, you can find your keys in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.

```